internal/services/containers/container_registry_resource.go

```diff
@@ -385,6 +391,16 @@ func resourceContainerRegistry() *pluginsdk.Resource {
 				return fmt.Errorf("ACR trust policy can only be applied when using the Premium Sku. If you are downgrading from a Premium SKU please set trust_policy {}")
 			}
 
+			exportPolicyEnabled := d.Get("export_policy_enabled").(bool)
+			if !exportPolicyEnabled {
+				if !strings.EqualFold(sku, string(containerregistry.SkuNamePremium)) {
+					return fmt.Errorf("ACR export policy can only be disabled when using the Premium Sku. If you are downgrading from a Premium SKU please unset `export_policy_enabled` or set `export_policy_enabled = true`")
+				}
```

Comment:
this seems reasonable for the user to do in a single apply, but we may need to call the Update API twice to make that change?

Reply:
@tombuildsstuff Is that covered by the test case [TestAccContainerRegistry_policies](https://github.com/hashicorp/terraform-provider-azurerm/blob/1f38fda7743d6319f1f231666468c72399d400f0/internal/services/containers/container_registry_resource_test.go#L450), where one update is enough?